modules/navigation/godot_navigation_server.cpp

```diff
@@ -143,7 +143,7 @@ COMMAND_2(map_set_active, RID, p_map, bool, p_active) {
 		}
 	} else {
 		int map_index = active_maps.find(map);
-		ERR_FAIL_COND(map_index < 0);
+		ERR_FAIL_INDEX(map_index, (int)active_maps.size());
```

Comment:
I'm not sure this can even be more than `size()`, that would be a bug in `LocalVector::find()`. The previous test was meant to check if the element wasn't found (so it's `-1`). So I think it would be better to use `ERR_FAIL_COND(map_index == -1)`.